### DISCUSSION TITLE
Bedwars1058 integration / ideas?

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#55] [Ideas]
This is a suggestion that can be considered after main plugin development is almost finished
So if a kit has a specific tag it will send request to bedwars1058 making the queuing players join a specified custom configured arena for 1v1's or 2v2
Stats will be saved in eden only
This would be very unique and help servers be more creative
Of course this is just a suggestion which I know won't be accepted anytime soon but this is something I had in mind

If not then let us execute custom commands by clicking on a kit to join bedwars game or something like that
In short you either give us bw1058 support
Or make inventory much more customisable with the ability to add custom non clickable slots, custom commands slot and of course kit selection (automatic)

**Comment by HierarchyMan** [comment]
@GoodestEnglish

**Comment by GoodestEnglish** [comment]
Won't work on this. This requires huge sync between server, player data and bungee. It is possible but it requires a lot of dev time which I don't have. You will need a external developer to work on this.

**Comment by HierarchyMan** [comment]
And what about the inventory system I mentioned?

**Comment by GoodestEnglish** [comment]
Could be something which I can add after I rework on the kit game rules menu
